### PR TITLE
QUIC constant

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

**Comment by yschimke** [comment]
Related to https://github.com/square/okhttp/issues/3683

In case we want to lang this in a next release, so interceptor doesn't have to wait for potentially another month.

But no rush either.

File: okhttp/src/main/java/okhttp3/Protocol.java [hunk 1]
```diff
@@ -78,12 +85,15 @@ public static Protocol get(String protocol) throws IOException {
     if (protocol.equals(HTTP_1_1.protocol)) return HTTP_1_1;
     if (protocol.equals(HTTP_2.protocol)) return HTTP_2;
     if (protocol.equals(SPDY_3.protocol)) return SPDY_3;
+    if (protocol.equals(QUIC.protocol)) return QUIC;
```
Comment (swankjesse) [inline okhttp/src/main/java/okhttp3/Protocol.java]: I wonder if we need this. This method is used for ALPN, and QUIC doesn't use ALPN. In particular, the string "quic" isn't used on the we wire anywhere, is it?

File: okhttp/src/main/java/okhttp3/Protocol.java [hunk 2]
```diff
@@ -78,12 +85,15 @@ public static Protocol get(String protocol) throws IOException {
...
     throw new IOException("Unexpected protocol: " + protocol);
   }
 
   /**
    * Returns the string used to identify this protocol for ALPN, like "http/1.1", "spdy/3.1" or
    * "h2".
+   *
+   * @link https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml
```
Comment (swankjesse) [inline okhttp/src/main/java/okhttp3/Protocol.java]: Neat. I'm surprised spdy/3.1 is absent.

File: okhttp/src/main/java/okhttp3/Protocol.java (hunk 1 above)
Comment (yschimke) [reply okhttp/src/main/java/okhttp3/Protocol.java]: It's useful as API

https://github.com/yschimke/oksocial/blob/master/src/main/kotlin/com/baulsupp/oksocial/commands/CommandLineClient.kt#L450

Comment (swankjesse) [reply okhttp/src/main/java/okhttp3/Protocol.java]: Works for me.

**Pull request merged**